css/general.css

```diff
@@ -32,3 +32,28 @@ ul {
   margin: 0 auto;
   padding: 0 3.2rem;
 }
+
+.btn,
+.btn:link {
+  font-size: 2rem;
+  font-weight: 500;
+  text-decoration: none;
+  background-color: var(--secondary-color);
+  color: #fff;
+  padding: 1.2rem 2.4rem;
+  justify-content: center;
+  border-radius: 4px;
+  cursor: pointer;
+  transition: all 0.3s;
+  border: none;
+}
+
+.btn:hover,
+.btn:active {
```

Comment:
As the previous reviewer requested, please add pressed state according to the Figma template. Here you are giving the same value for pressed and hover state. And the color codes doesn't match with the Figma template. Please fix the color of active/pressed state to ##008552. 
[OPTIONAL] Kindly fix the color of hover state of this button to #36B37F. Also `box-shadow: 0px 8px 16px rgba(54, 179, 127, 0.24);`